### PR TITLE
Add accNames to all progress bar examples

### LINKED ISSUE #36714: Progress Bar Examples have accessibility issues
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

The provided examples at: https://getbootstrap.com/docs/5.2/components/progress/ have accessibility issues reported by both Google Lighthouse and Webhint.io

I think that the progress bar should have a non-empty `aria-label` attribute or a `title`.

REF:
https://dequeuniversity.com/rules/axe/4.4/aria-progressbar-name?application=axeAPI

Let me know what you think, I'd be happy to open a PR...

(I realise this might be flagged as an accessibility issue incorrectly and maybe the `aria-valuenow`, `aria-valuemax` and `aria-valuetext` attributes are enough? cc @patrickhlauke )

### Reduced test cases

Webhint screenshot:
image

Lighthouse screenshot:
image

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

### What version of Bootstrap are you using?

v5.2.0-beta1

### PR DESCRIPTION
**Pull request by patrickhlauke**
* also expands advice in progress.md, explicitly mentions giving them an accName, and includes the callout about using color to convey meaning
* updates cheat sheets, but note that for the RTL one, my arabic is...lacking. we should get somebody who knows what they're doing to translate those `aria-label`s properly

Closes #36714

### CODE REVIEW

File: site/content/docs/5.2/examples/cheatsheet-rtl/index.html [hunk 1]
```diff
@@ -1389,26 +1389,26 @@ <h3>شريط التقدم</h3>
       <div>
         {{< example show_markup="false" >}}
         <div class="progress mb-3">
-          <div class="progress-bar" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100">0%</div>
+          <div class="progress-bar" role="progressbar" aria-label="Example with label" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100">0%</div>
         </div>
         <div class="progress mb-3">
-          <div class="progress-bar bg-success w-25" role="progressbar" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100">25%</div>
+          <div class="progress-bar bg-success w-25" aria-label="Success example with label" role="progressbar" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100">25%</div>
```
Comment (julien-deramond) [inline site/content/docs/5.2/examples/cheatsheet-rtl/index.html:1395]:
```suggestion
          <div class="progress-bar bg-success w-25" role="progressbar" aria-label="Success example with label" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100">25%</div>
```
Was OK but just for consistency.

**Review by julien-deramond — CHANGES_REQUESTED** [review]
LGTM! Thanks for this PR @patrickhlauke.
Just a small change proposed for consistency.
Regarding the arabic version (RTL) we could maybe ask in the Slack general channel for some translation if anyone can help use in the community :shrug:

**Review by mdo — APPROVED** [review]

**Review by julien-deramond — APPROVED** [review]

**Comment by julien-deramond** [comment]
> Can I paste the translations here? @julien-deramond

Yes sure. Thanks a lot <3

**Comment by AqeelAT** [comment]
- "Example with label" -> "مثال مع عنوان"
- "Success example with label" -> "مثال ناجح مع عنوان"
- "Info example with label" -> "مثال توضيح مع عنوان"
- "Warning example with label" -> "مثال تنبيه مع عنوان"
- "Danger example with label" -> "مثال خطر مع عنوان"
- "Segment one - default example" -> "القسم الأول - مثال افتراضي"
- "Segment two - animated striped success example" -> "القسم الثاني - مثال ناجح مقلّم متحرك"

Sorry for the delay.
Let me know if there is anything else I can do to help with bootstrap's RTLization.

@julien-deramond @mdo

**Comment by patrickhlauke** [comment]
wonderful, thank you @AqeelAT

**Pull request merged**